src/components/popover/__snapshots__/popover.test.jsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Popover has defaults 1`] = `
```

Comment:
I think it might make sense to take a snapshot when it is opened as well. I don't wanna get too crazy with snapshotting but it sounds reasonable for something this the core functionality and a boolean.

Reply:
Done and done, with a new props API `startOpen`.